### ISSUE TITLE
[Anon] Bug report: Warhammer 40,000 8th Edition.gstz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k#7146]
**File:** Warhammer 40,000 8th Edition.gstz

**BattleScribe version:** 2.03.13

**Platform:** Windows

**Dropbox:** Yes

**Description:** When adding Wolf Pack with Terminator Armour units from Space Wolves and giving them the Cyclone Missile Launcher and Storm Bolter option, there is no option to switch out the Storm Bolter as indicated in the Soace Wolves Codex Errata 1.4.

**Comment by Mad-Spy** [comment]
You can give any Terminator in the unit a Cyclone Missile Launcher in order to achieve this. 

You only need to use the HW Terminator if you actually want to give them a Heavy weapon.

**Comment by tmwoodsy** [comment]
Hey Mad-Spy.  I wrote this a little tired last night.  I just want to be sure we are clear, because I still don't know that the issue was addressed. In the Errata attached, on page 3, second question:
"Q: If I replace a Wolf Guard Terminator’s storm bolter with a
cyclone missile launcher and storm bolter from the Terminator
Heavy Weapons list (as per the third bullet point), can I
subsequently replace the new storm bolter with an item from the
Combi-weapons or Terminator Melee Weapons list (as per
the second bullet point)?
A: Yes."

Per this understanding, once I have added a Wolf Guard in Terminator Armour with Heavy Weapons, and have selected the Cyclone Missile Launcher with Storm Bolter as the heavy weapon, shouldn't I be able to then switch out the Storm Bolter for a Combi-weapon or Space Wolf Terminator Melee Weapon?

[warhammer_40000_space_wolves_en-1.pdf](https://github.com/BSData/wh40k/files/4658101/warhammer_40000_space_wolves_en-1.pdf)
